mod_reforged/config/item_names.nut

```diff
@@ -1,3 +1,52 @@
+// Vanilla Adjustments
+::MSU.Array.removeValues(::Const.Strings.CrossbowNames, [
+	"Betrayer",		// moved over to dagger names
+]);
+::Const.Strings.CrossbowNames.extend([
```

Comment:
More cool crossbow names:
- Thunderbolt
- Ravenshrike
- Bloodbolt
- Grimshot
- Wyrmfang
- Blackthorn
- Thunderclap
- Stormbreaker
- Steelwind

Reply:
"Thunderclap" feels more fitting for the handgonne so I do not include it